bin/test

```diff
@@ -8,3 +8,24 @@ xcrun xcodebuild \
   test \
   | xcpretty --color
 
+xcrun xcodebuild \
+  -workspace Argo.xcworkspace \
+  -scheme Argo-iOS \
+  -destination "platform=iOS Simulator,name=iPhone 6" \
+  test \
+  | xcpretty --color
+
+xcrun xcodebuild \
+  -workspace Argo.xcworkspace \
+  -scheme Argo-watchOS \
+  -destination "generic/platform=watchOS" \
+  CODE_SIGNING_REQUIRED=NO \
+  CODE_SIGN_IDENTITY="" \
```

Comment:
Right. There's no such thing as a test bundle that will run again a watchOS framework. Fun stuff. Making sure it builds seems like a reasonable alternative for now. Basically, I want to make sure CI catches people not adding files to all of the frameworks.